config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/environment/EnvironmentBundleCreator.java

```diff
@@ -70,11 +72,12 @@ public Bundle createEnvironmentBundle(Map<String, String> environmentProperties,
         final DocumentBuilder documentBuilder = documentTools.getDocumentBuilder();
         final Document document = documentBuilder.newDocument();
 
+        // Passing Bundle name and version string with config env name (<name>-<version>-*env) as project name
         Map<String, BundleArtifacts> bundleElements = bundleEntityBuilder.build(environmentBundle,
-                EntityBuilder.BundleType.ENVIRONMENT, document, bundleFileName, "", null);
+                EntityBuilder.BundleType.ENVIRONMENT, document, new ProjectInfo(bundleFileName, EMPTY, EMPTY));
         for (Map.Entry<String, BundleArtifacts> entry : bundleElements.entrySet()) {
```

Comment:
if key is not used, just iterate values instead of entrySet

Reply:
Done!